### DISCUSSION TITLE
GitHub Desktop assignment

### DISCUSSION THREAD

**Discussion by vmwielgus** [ua-gist604b-f24/syllabus#6] [General]
Hi, I'm not sure if I used the right repo in the GitHub Desktop assignment. I can see that my ,png ended up in the browser. But I received an error message in my email. Would anyone be able to tell me if I did this assignment correctly? In the meantime I've moved on, but feeling comfortable navigating GitHub is important to me. Thanks for any help you can give.
![github_desktop](https://github.com/user-attachments/assets/0d550240-30d4-4eeb-b44c-10366ec7e674)
![ci_,master_fail](https://github.com/user-attachments/assets/edc072b4-6aa1-45cd-99ba-a06601682b91)

**Comment by aaryno** [comment]
Violet, I emailed you about this. The issue here is that you were Finished (10/10) when you had the Open Pull Request. However, you merged the Pull Request. I wish I could automatically protect the master branch so this wouldn't happen so you will have to just be aware that I am looking for Open Pull Requests for all assignments from Module 1 onward.
